### PR TITLE
Fail CI if change in autogenerated files are detected

### PR DESCRIPTION
**Pull request by Kaitou786**

### CODE REVIEW

**Comment by Kaitou786** [comment]
There were some auto-generated files changes needed to be committed to our main branch, I saw when the CI failed, so for that, I created #165. This commit, for now, have been rebased with #165 for verification, but those files from this PR will be removed when we merge that commit

File: hack/go-vet.sh [hunk 1]
```diff
@@ -19,10 +19,10 @@ GOPATH=$(go env GOPATH)
 ./hack/addheaders.sh
 
 operator-sdk generate k8s
-operator-sdk generate crds
+operator-sdk generate crds --crd-version=v1beta1
```
Comment (ricardozanini) [inline hack/go-vet.sh:22]: :clap:

**Review by ricardozanini — APPROVED** [review]

File: hack/go-vet.sh (hunk 1 above)
Comment (LCaparelli) [reply hack/go-vet.sh:22]: nice catch o/

File: hack/go-vet.sh [hunk 2]
```diff
@@ -19,10 +19,10 @@ GOPATH=$(go env GOPATH)
...
 
 # get the openapi binary
-command -v openapi-gen >/dev/null || go build -o $GOPATH/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen
+command -v openapi-gen >/dev/null || go build -o $GOPATH/bin/openapi-gen k8s.io/kube-openapi/cmd/openapi-gen
```
Comment (LCaparelli) [inline hack/go-vet.sh:25]: Nice catch v2

**Review by LCaparelli — APPROVED** [review]

**Pull request merged**